examples/CustomEntitlementComputationSample/gradle/libs.versions.toml

```diff
@@ -33,5 +33,4 @@ navigation-compose = "androidx.navigation:navigation-compose:2.5.3"
 compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling" }
 compose-ui-test-manifest = { module = "androidx.compose.ui:ui-test-manifest" }
 
-revenuecat = { module = "com.revenuecat.purchases:purchases", version.ref = "purchases" }
-revenuecat-amazon = { module = "com.revenuecat.purchases:purchases-store-amazon", version.ref = "purchases" }
+revenuecat = { module = "com.revenuecat.purchases:purchases@customEntitlementComputation", version.ref = "purchases" }
```

Comment:
We should update this to use the correct package once it's deployed... I guess that should be in a different PR

Reply:
I forgot to update it here. I just did in [883202b](https://github.com/RevenueCat/purchases-android/pull/1167/commits/883202b3eee18e4375876d625aad4505fb0f4023)